pkg/commands/rules.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/grafana/cortextool/pkg/rules"
 	log "github.com/sirupsen/logrus"
 	"gopkg.in/alecthomas/kingpin.v2"
-	"gopkg.in/yaml.v2"
+	yamlv3 "gopkg.in/yaml.v3"
```

Comment:
YAML.v3, [has better indentation than v2](https://ubuntu.com/blog/api-v3-of-the-yaml-package-for-go-is-available) - this aligns with the output that jsonnet produces as we generate the rules files. In turn, this allows us to have a minimal diff from the original source thus having a better inspection.